src/Avalonia.Native/AvaloniaNativePlatformExtensions.cs

```diff
@@ -73,5 +73,10 @@ public class MacOSPlatformOptions
         /// You can prevent Avalonia from adding those items to the OSX Application Menu with this property. The default value is false.
         /// </summary>
         public bool DisableDefaultApplicationMenuItems { get; set; }
+        
+        /// <summary>
+        /// 
```

Comment:
Empty comment?

Reply:
It's usually a bad idea IMO to have options that disable something when setting them to `true`, but given that that option above does the same I guess this is at least internally consistent.

(What I mean is: ideally this would be called `EnableNativeMenus` and have a default value of `true`).

Lets at least add a comment here:

```suggestion
        /// Gets or sets a value indicating whether the native macOS menu bar will be enabled for the application.
```

Reply:
As an aside, looks like the [.NET naming guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#:~:text=DO%20name%20Boolean%20properties%20with%20an%20affirmative%20phrase%20(CanSeek%20instead%20of%20CantSeek).%20Optionally%2C%20you%20can%20also%20prefix%20Boolean%20properties%20with%20%22Is%22%2C%20%22Can%22%2C%20or%20%22Has%22%2C%20but%20only%20where%20it%20adds%20value.) agree with me ;)